ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # Set to True for GPU enabled serverless sessions/dataproc clusters
</pre><p>Set to True for GPU-enabled environments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I have actually removed it and rely on torch.cuda.is_available() only</p>